content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/start.rb

```diff
@@ -30,6 +31,17 @@ def service
                 end
               end
 
+              def check_interval
+                return unless @handle.root['ae_retry_ttl']
+
+                interval = @handle.root['ae_retry_ttl'] / @handle.root['ae_state_max_retries']
```

Comment:
We should ensure that both of these values are integers.  (`.to_i`)  The `ae_state_max_retries` value is an optional value for a state machine step, what value would we end up with here when it is blank?  Divide by zero?  💥